maildir2mbox.c

```diff
@@ -14,6 +15,8 @@
 #include "myctime.h"
 #include "maildir.h"
 
+extern int rename(const char *, const char *);
```

Comment:
Are you already plotting the demise of `puts()`, so we can come back for `<stdio.h>` on these?

Reply:
I would love to, but I think ```puts()``` is one of the things that is likely actually used by external patches.

Reply:
Yes, that’s true.

I’m not worried we’ll forget to come back to this someday (when external patches are way less common). Pretty soon, when we have hardly any `extern` left, this’ll stick out. :-)